auth-web/src/views/auth/create-account/DuplicateAccountWarningView.vue

```diff
@@ -66,23 +66,32 @@ export default class DuplicateAccountWarningView extends Vue {
     @OrgModule.State('currentOrganization') private currentOrganization!: Organization
     @OrgModule.Action('addOrgSettings') private addOrgSettings!: (currentOrganization: Organization) => Promise<UserSettings>
     @OrgModule.Action('syncOrganization') private syncOrganization!: (orgId: number) => Promise<Organization>
+    @UserModule.Action('getUserAccountSettings') private getUserAccountSettings!: () => Promise<any>
     private orgsOfUser: OrgWithAddress[] = []
     private isLoading: boolean = false
     @Prop({ default: '' }) redirectToUrl !: string
 
     private async mounted () {
+      if (!Array.isArray(this.currentUserAccountSettings) || !this.currentUserAccountSettings.length) {
```

Comment:
not sure if a this.currentUserAccountSettings?.length works with undefined?

Reply:
Good one. It catches undefined case as well :)